core/chains/evm/config/toml/config_test.go

```diff
@@ -0,0 +1,29 @@
+package toml_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/toml"
+)
+
+func Test_Validate_Defaults(t *testing.T) {
+	var err error
+	for _, id := range toml.DefaultIDs {
+		t.Run(fmt.Sprintf("chainID-%s", id), func(t *testing.T) {
+			evmCfg := &toml.EVMConfig{
+				ChainID: id,
+				Chain:   toml.Defaults(id),
+			}
+
+			err = multierr.Append(err, evmCfg.ValidateConfig())
+			err = multierr.Append(err, evmCfg.Chain.ValidateConfig())
+			err = multierr.Append(err, evmCfg.Chain.GasEstimator.ValidateConfig())
```

Comment:
```suggestion
			err = config.Validate(evmCfg)
```

Reply:
I'm actually trying to decouple evm config tests from core, so I won't be able to use this.

Reply:
I think this is a common helper that should probably be moved to chainlink-common.

Reply:
@jmank88 when I tried to move this to chainlink-common I noticed we have a depguard for `go.uber.org/multierr`. Is there an alternative we use instead?

Reply:
https://pkg.go.dev/errors#Join

Reply:
I noticed `Validate()` uses `MultiErrorList` method that requires multierr package to unwrap errors in a slice. Is this something we want to refactor separately?

Reply:
Yeah so this just came up for Solana. I think we have a way to fix it and use the std lib instead

Reply:
Not yet but we can sync up with @aalu1418